detox/src/devices/drivers/android/genycloud/GenyCloudDriver.js

```diff
@@ -0,0 +1,66 @@
+const _ = require('lodash');
+const AndroidDriver = require('../AndroidDriver');
+const GenyCloudExec = require('./exec/GenyCloudExec');
+const RecipesService = require('./services/GenyRecipesService');
```

Comment:
What's stopping you from renaming the files as well to `RecipesService`, `InstanceLookupService`, `InstanceLifecycleService`, `InstanceNaming`, `AllocationHelper`, `DeviceQueryHelper` correspondingly? Everything is contained in `genymotion/` folder, so it just should be 100% fine.

Reply:
Otherwise, that looks a bit inconsistent, to my taste. 🤷